### PR TITLE
Fix the spamming log in CLI.

### PR DESCRIPTION
**Pull request by larry-aptos**
## Type of Change
- [ ] New feature
- [x] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [ ] Refactoring
- [ ] Dependency update
- [ ] Documentation update
- [ ] Tests

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [ ] Full Node (API, Indexer, etc.)
- [ ] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [x] Aptos CLI/SDK
- [ ] Developer Infrastructure
- [ ] Other (specify)

## How Has This Been Tested?

Locally tested and it looks good. 

image

## Checklist
- [x] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I identified and added all stakeholders and component owners affected by this change as reviewers
- [ ] I tested both happy and unhappy path of the functionality
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by larry-aptos** [comment]
* **#14709** <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-core/14709?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @larry-aptos and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by just-in-chang — APPROVED** [review]

**Review by banool — APPROVED** [review]
Update changelog please.

**Review by yuunlimm — APPROVED** [review]

File: crates/aptos/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ All notable changes to the Aptos CLI will be captured in this file. This project
 
 ## Unreleased
 
+## [4.2.2] - 2024/09/20
+- Fix localnet indexer processors that were emitting spamming logs in 4.2.1.
+
```
Comment (gregnazario) [inline crates/aptos/CHANGELOG.md:9]: Can you update `crates/aptos/Cargo.toml` for the new version

Comment (larry-aptos) [reply crates/aptos/CHANGELOG.md:9]: done.

**Pull request merged**